included/9.2.0/Dockerfile

```
@@ -0,0 +1,69 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 9.2.0 cypress/browsers:node16.5.0-chrome94ff93
+#
+# build this image with command
+#   docker build -t cypress/included:9.2.0 .
+#
+FROM cypress/browsers:node16.5.0-chrome94ff93
```

I think the image tag here might be wrong? I can't find it in [dockerhub](https://hub.docker.com/r/cypress/browsers/tags) but I can find the suggestion below. Also looks to be what other previous images are based off of
```suggestion
FROM cypress/browsers:node16.5.0-chrome94-ff93
```